src/test/java/selenium/Locators.java

```diff
@@ -0,0 +1,21 @@
+package selenium;
+
+import org.openqa.selenium.By;
+
+public class Locators {
+    public static By xpath = By.xpath("//input[@id='user-name']");
+    public static By id = By.id("user-name");
+    public static By cssSelector = By.cssSelector("#user-name");
+    public static By userName = By.name("user-name");
+    public static By className = By.className(".title");
+    public static By tagName = By.tagName("form");
+    //The following locators could be found on Products page
+    public static By partialLinkText = By.partialLinkText("Backpack");
+    public static By LinkText = By.linkText("Sauce Labs Backpack");
```

Comment:
better to make these fields as final static and don't forget about Java naming convention

Reply:
Done